app/services/Capi.scala

```diff
@@ -88,13 +88,65 @@ class GuardianCapi(config: ApplicationConfiguration)(implicit ex: ExecutionConte
     query
   }
 
+  // Sadly there's no easy way of converting a CAPI client response into JSON so we'll just proxy - similar to controllers.FaciaContentApiProxy
+  def getPrefillArticles(issueDate: ZonedDateTime, capiPrefillQuery: CapiPrefillQuery, currentPageCodes: List[String]): Future[SearchResponse] = {
+    val fields = List(
+      "newspaperEditionDate",
```

Comment:
should these be kebab-cased?

Reply:
They were previously kebab-case and *seemed* to work, but the docs use camelCase and it also *seems* to work

Reply:
Except in the `useDate` case, which MUST be kebab case lols.